src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -619,6 +623,10 @@ export class StudyViewPageStore
         { deep: false }
     );
 
+    @action public persistNewlyCreatedGroup(groupId: string): void {
```

Comment:
I'm not sure I understand the word "persist" here, it doesn't look like its persisting it, just keeping track of it as created

Reply:
Yes, I'll rename it `trackNewlyCreatedGroup`